components/tabs/Reuse/ReuseTabsService.cs

```diff
@@ -176,6 +176,50 @@ private void AddReuseTabsPageItem(Type pageType)
             _pageMap[url] = reuseTabsPageItem;
         }
 
+
+          
+     
+        public void AddReuseTabsPageItem(string pageUrl, string? routeUrl, RenderFragment? title)
```

Comment:
Can we merge these two urls into one? I don't think there are any scenarios that need to be used together.

Reply:
It is used when a page needs to input parameters and loads different components according to different parameters, that is, the page is public. I'm making a low code platform here, and basically all forms are drag-and-drop configurations, so they all share a page and load different form configurations according to different menus.
For example, in my example above, the route setting of my public page is /module/{group}/{menu}, if you search by the actual URL /module/1000/1001,

var routeAttribute = w.GetCustomAttribute<RouteAttribute>();
var reuseTabsAttribute = w.GetCustomAttribute<ReuseTabsPageAttribute>();
var url = reuseTabsAttribute?. PinUrl ?? routeAttribute?. Template;

In this case, the url is /module/{group}/{menu}, and the corresponding pageType cannot be found.

Reply:
I konw the route template. But I think this method just adds tabs rather than actually accessing the page. Until the user hits the tab, blazor finds the corresponding page type through PageUrl(with route parameters) and renders it.

Reply:
> But the question is, how to find the corresponding pageType according to /module/1000/1001? I don't have a good plan.

In theory, when you click the tab, it would navigate to the target url and RouteView would find the PageType and route parameters.

Reply:
Then it should call the ReuseTabsService.TrySetRouteData method and build the ReuseTabsPageItem.
https://github.com/ant-design-blazor/ant-design-blazor/pull/3671/files/e2b47e83ccb4cd63bb6f91978fae7b4e1a9cf7c3#diff-902b04cffed819dd7d81d3579281070c11cef725b98e487d5add8f571010cf2cR56-R81